agency-api-spec.md

```diff
@@ -221,3 +333,6 @@ Services like [api.data.gov](https://api.data.gov/about/) provide this authentic
 	}
```

Comment:
The sample call is looking good--thanks for taking the time to add a realistic payload. Do you think the agency-specific fields should also be a part of this sample?